docs/developing-locally-docker.rst

```diff
@@ -167,16 +167,18 @@ docker
 
 The ``container_name`` from the yml file can be used to check on containers with docker commands, for example: ::
 
-    $ docker logs worker
-    $ docker top worker
+    $ docker logs <project_slug>_local_celeryworker
```

Comment:
Actually, cookiecutter will replace the expression {{ cookiecutter.project_slug }} for the name of the project.

<project_slug> will just remain the same in the generated project. 

Please replace the <> one fot he {{}} one.

Reply:
Not here, actually. That's the template docs, not the generated project ones.

Reply:
Oh! You are correct. My mistake!